src/boot/bootctl-uki.c

```diff
@@ -143,21 +144,47 @@ static int read_pe_section(FILE *uki, const struct PeSectionHeader *section,
         return 0;
 }
 
+static void inspect_osrel(char *osrel, size_t osrel_size) {
+        _cleanup_fclose_ FILE *s = NULL;
+        _cleanup_free_ char *pname = NULL, *name = NULL;
+        int r;
+
+        assert(osrel);
+        s = fmemopen(osrel, osrel_size, "r");
+        if (!s)
+                return (void)log_warning_errno(errno, "Failed to open embedded os-release file, ignoring: %m");
```

Comment:
Please use `(void) log…` (with a space). It's like that everywhere else in the codebase.